fast64_internal/oot/oot_level_writer.py

```diff
@@ -349,8 +349,11 @@ def readRoomData(room, roomHeader, alternateRoomHeaders):
     room.disableSunMoon = roomHeader.disableSunMoon
     room.echo = roomHeader.echo
     room.objectList.extend([getCustomProperty(item, "objectID") for item in roomHeader.objectList])
-    if len(room.objectList) > 15:
-        raise PluginError("Error: A scene can only have a maximum of 15 objects (OOT, not blender objects).")
+
+    if len(room.objectList) > 16:
+        print(
+            "Warning: A room can only have a maximum of 16 objects in its object list, unless more memory is allocated in code.",
+        )
```

Comment:
so the limit is actually 19, though 16 is a good assumption as long as you assume that the "normal" scenario of gameplay_keep/player/subkeep are there

If it wouldnt be annoying to implement, it may be good to account for those other 3 (which are set via other commands) isntead of implying the limit in the code is directly set to 16. At the very least, it may confuse someone who then goes into to expand this slot count when they are expecting 16 but see 19.
But yea, not the biggest deal

Reply:
The issue is that blender doesn't do word wrap on labels so its difficult to add more info without truncating the text and making it unreadable. Ideally I would make a warning flash on export, with a checkbox to disable that warning, but the way blender works makes it really difficult to flash warnings without stopping the operator execution.

Reply:
yea my comment is less so about adding more info the warning, but possibly making the check more accurate. an implied limit of 16 will get the job done, but if we wanted it to be more precise, it can check if link/subkeep are present from the other commands in the room that supply them